### PR TITLE
Remove fullpage.js

### PR DESCRIPTION
**Pull request by c-w**
Resolves https://github.com/ascoderu/ascoderu.github.io/issues/17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,5 +3,3 @@
 [![Travis CI status](https://api.travis-ci.org/ascoderu/ascoderu.github.io.svg?branch=development)](https://travis-ci.org/ascoderu/ascoderu.github.io)
 
 Public information site for Ascoderu, hosted via Github-Pages and Cloudflare at [ascoderu.ca](https://ascoderu.ca).
-
-Built with [Bootstrap](http://getbootstrap.com/docs/3.3/) and [FullPage](https://alvarotrigo.com/fullPage/).
```
Comment (laura-barluzzi) [inline README.md]: You should remove only the part of FullPage since we still use Bootstrap

File: src/index.html [hunk 2]
```diff
@@ -15,15 +15,12 @@
...
   <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fullPage.js/2.8.9/jquery.fullPage.min.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap3-dialog/1.34.7/css/bootstrap-dialog.min.css">
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
 
-  <div id="website-content" class="hidden">
-
```
Comment (laura-barluzzi) [inline src/index.html]: Restore this because it was fixing the flickering on load

Comment (c-w) [reply src/index.html]: Done in b827b8e.

File: README.md (hunk 1 above)
Comment (c-w) [reply README.md]: Done in ef5b029.

**Pull request merged**